src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -77,6 +82,7 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(Items.SNOW).grist(COBALT, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.ICE).grist(COBALT, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(Items.PACKED_ICE).grist(BUILD, 10).grist(COBALT, 6).buildFor(recipeSaver, Minestuck.MOD_ID);
+		GristCostRecipeBuilder.of(Items.POWDER_SNOW_BUCKET).grist(BUILD, 16).grist(CHALK, 4).grist(RUST, 3).buildFor(recipeSaver, Minestuck.MOD_ID);
```

Comment:
Move this to around line 315, where we use ContainerGristCostBuilder to determine the cost of each vanilla bucket's contents

Reply:
Now use the ContainerGristCostBuilder instead of GristCostRecipeBuilder